e2e/commands/test.e2e.js

```diff
@@ -66,6 +66,11 @@ describe('bit test command', function () {
       const output = helper.testComponent('utils/is-type');
       expect(output).to.have.string('tests passed');
     });
+    it('Should not be able to run tests with wrong tester env', () => {
+      helper.importTester('bit.envs/testers/jest');
```

Comment:
- [ ] add the tester to fixture (similar to babel) then don't pass the tester
- [ ] you might also search for other places we pass testers and if they use the same tester you add to fixture just remove the id as well

Reply:
this requires further changes to be made in the helper.e2e.js file to allow to use local tester. for now I've used remote testing envs.